components/centraldashboard/frontend/main-page.html

```diff
@@ -135,14 +145,29 @@
                 <neon-animated-pages selected="[[homeOrIframeViewIndex]]" class="flex layout vertical" entry-animation="fade-in-animation" exit-animation="fade-out-animation">
                     <neon-animated-pages id='PrimaryView' selected="[[primaryViewIndex]]" class="flex layout vertical" entry-animation="fade-in-animation" exit-animation="fade-out-animation">
                         <article id='Dashboard'>
-                            <paper-card heading="Getting Started">
+                            <paper-card heading="Getting Started" class="Getting-Started">
                                 <template is="dom-repeat" items="[[gettingStartedItems]]">
                                     <paper-icon-item>
                                         <iron-icon icon="[[item.icon]]" slot="item-icon"></iron-icon>
-                                        [[item.text]]
+                                        <paper-item-body two-line>
+                                            <div>[[item.text]]</div>
+                                            <aside secondary>Lorem ipsum...</aside>
+                                        </paper-item-body>
                                     </paper-icon-item>
                                 </template>
                             </paper-card>
+                            <paper-card heading="Quick Links" class="Quick-Links">
+                                <template is="dom-repeat" items="[[quickLinks]]">
+                                    <article class="link">
+                                        <paper-item-body>[[item.text]]</paper-item-body>
+                                        <paper-icon-button class="button" icon='arrow-forward' alt='[[item.text]]', on-click='openQuickLink'></paper-icon-button>
+                                    </article>
+                                </template>
+                                <article class="link more-coming">
+                                    <paper-item-body>More coming soon</paper-item-body>
+                                    <paper-icon-button class="button" icon='arrow-forward' diabled></paper-icon-button>
+                                </article>
+                            </paper-card>
                         </article>
                         <article id='Activity'></article>
```

Comment:
Do we have anything to show here? If not, perhaps we should hide this tab until we do?

Reply:
Done!